src/routes/relay/relay.controller.spec.ts

```diff
@@ -69,32 +69,30 @@ describe('RelayController', () => {
       it('should return a 201 when the body is valid', async () => {
         axios.default.get = jest.fn().mockResolvedValue({ data: 'mockSafe' });
 
-        const sponsorServiceSpy = jest
-          .spyOn(mockSponsorService, 'sponsoredCall')
-          .mockImplementation(() => Promise.resolve({ taskId: '123' }));
+        (mockSponsorService.sponsoredCall as jest.Mock).mockImplementation(() =>
```

Comment:
If you remove the type from the `mockSponsorService` then this should work without any casts. I think it should be fine since this is a testing scenario but up to you! 👍 

```typescript
let mockSponsorService: ISponsorService;
                        ^^^^^^^^^^^^^^^
                        |||||||||||||||
                   remove this or `unknown`
```

Reply:
I'll leave it as is for now to keep it somewhat type safe. It would be nice if `jest.Mock` was a generic as this seems to be a common issue from what I can find online.